hw-vendor/marvell/README.md

```diff
@@ -0,0 +1,116 @@
+# Marvell Comm Channel
+
+Welcome to the Marvell Comm Channel project! This guide provides a step-by-step walkthrough to compile, build, and launch the components of the code.
+
+## Table of Contents
+
+- [Compilation Steps](#compilation-steps)
+- [Docker Build Steps](#docker-build-steps)
+- [Launch Steps](#launch-steps)
+- [Steps to Enable IPv6 Link Local](#steps-to-enable-ipv6-link-local)
+- [License](#license)
+
+## Compilation Steps
+
+Follow these steps to compile the Marvell Comm Channel code:
+
+1. **Install Dependencies:**
+
+    ```sh
+    go mod vendor
+    ```
+
+2. **Navigate to the Marvell Directory:**
+
+    ```sh
+    cd hw-vendor/marvell
+    ```
+
+3. **Build the Project:**
+
+    ```sh
+    go build -o mrvl-commchannel
+    ```
+
+## Docker Build Steps
+
+To create and push a Docker image:
+
+1. **Build the Docker Image:**
+
+    ```sh
+    docker build -f Dockerfile.daemon.rhel .
+    ```
+
+2. **Tag the Docker Image:**
+
+    ```sh
+    docker tag <image_id> <TagName:latest>
+    ```
+
+3. **Push the Docker Image:**
+
+    ```sh
+    docker push <TagName:latest>
+    ```
+
+## Launch Steps
+
+### Steps to Enable IPv6 Link Local
+
+1. **Edit Network Connection:**
+
+    ```sh
+    nmcli con edit type ethernet con-name ens81
```

Comment:
Shouldn't the VSP manage these connections? I.e. the VSP should be self contained and should require outside help from the controller/user for setting up the interfaces.